client/src/plugins/report-feedback/ReportFeedback.js

```diff
@@ -100,7 +100,7 @@ export class ReportFeedback extends PureComponent {
 
     return (
       <Fragment>
-        <Fill slot="status-bar__app" group="9_report-feedback">
+        <Fill slot="status-bar__app" group="9_feedback">
```

Comment:
Is there a reason for this change?

Reply:
We use `priority`>  `group`. The priority can only be used in the same group and the groups are sorted alphabetically. Since these icons are all in separate groups - and we can't start doing `10_....` (because 1 < 9) to order them - then we we can either put the right side icons all in the same group like `9_app-icons` and give them different priorities (in my opinion, the better way to do this). Or find a hack like renaming the group to be alphabetically first, like I did.